tidb-cloud/config-s3-and-gcs-access.md

```diff
@@ -99,35 +99,47 @@ To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the
+
+        > **Tip:**
+        >
+        > Alternatively, you can also click the name of your cluster on the **Clusters** page and click **Import Data** in the **Import** area.
+
+    3. On the **Data Import** page, click **Show Google Cloud Service Account ID**, and then copy the Service Account ID for later use.
+
+2. In the Google Cloud Platform (GCP) Management Console, create an IAM role for your GCS bucket.
 
-    2. Click **Show Google Cloud Service Account ID**, and then copy the Service Account ID.
+    1. Sign in to the [GCP Management Console](https://console.cloud.google.com/).
+    2. Go to the [Roles](https://console.cloud.google.com/iam-admin/roles) page, and then click **CREATE ROLE**.
 
-2. In the Google Cloud Platform (GCP) Management Console, go to **IAM & Admin** > **Roles**, and then check whether a role with the following read-only permissions of the storage container exists.
+        ![Create a role](/media/tidb-cloud/gcp-create-role.png)
 
-    - storage.buckets.get
-    - storage.objects.get
-    - storage.objects.list
+    3. Enter a name, description, ID, and role launch stage for the role. The role name cannot be changed after the role is created.
+    4. Click **ADD PERMISSIONS**.
+    5. Add the following read-only permissions to the role, and then click **Add**.
 
-    If yes, you can use the matched role for the target TiDB cluster in the following steps. If not, go to **IAM & Admin** > **Roles** > **CREATE ROLE** to define a role for the target TiDB cluster.
+        - storage.buckets.get
+        - storage.objects.get
+        - storage.objects.list
 
-3. Go to **Cloud Storage** > **Browser**, select the GCS bucket you want TiDB Cloud to access, and then click **SHOW INFO PANEL**.
+        ![Add permissions](/media/tidb-cloud/gcp-add-permissions.png)
 
-    The panel is displayed.
+3. Go to the [Bucket](https://console.cloud.google.com/storage/browser) page, click the name of the GCS bucket you want TiDB Cloud to access.
 
-4. In the panel, click **ADD PRINCIPAL**.
+4. On the **Bucket details** page, click the **PERMISSIONS** tab, and then click **GRANT ACCESS**.
 
-    The dialog box for adding principals is displayed.
+    ![Grant Access to the bucket ](/media/tidb-cloud/gcp-bucket-permissions.png)
 
-5. In the dialog box, perform the following steps:
+5. Fill in the following information to grant access to your bucket, and then click **SAVE**.
 
-    1. In the **New Principals** field, paste the Google Cloud Service Account ID of the target TiDB cluster.
-    2. In the **Role** drop-down list, choose the role of the target TiDB cluster.
-    3. Click **SAVE**.
+    - In the **New Principals** field, paste the Google Cloud Service Account ID of the target TiDB cluster.
+    - In the **Select a role** drop-down list, choose the IAM role you just created.
```

Comment:
There is no corresponding custom roles  will be displayed if the user filter in the drop down list directly ,it is better to notice users they should  click "custom" ,then the created role will be displayed .